dist/index.js

```diff
@@ -6820,7 +6820,7 @@ function outside (version, range, hilo, options) {
       throw new TypeError('Must provide a hilo val of "<" or ">"')
   }
 
-  // If it satisifes the range it is not outside
```

Comment:
This appears to come from: https://github.com/npm/node-semver/blob/f56505b1c08856a7e6139f6ee5d4580f5f2feed8/ranges/outside.js#L35